docs/apidiffs/current_vs_latest/opentelemetry-exporter-zipkin.txt

```diff
@@ -1,2 +1,12 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
++++  NEW CLASS: PUBLIC(+) FINAL(+) io.opentelemetry.exporter.zipkin.OtelToZipkinSpanTransformer  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW INTERFACE: java.util.function.Function
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.logging.Logger logger
```

Comment:
let's not expose the logger here! We got bit by that with the exporter logger itself; no need to repeat that mistake. :)

Reply:
🙈 Whoops! Imagine someone intentionally using that tho...... 🤯 
Will fix.

Reply:
yeah, probably no one will, but now we're stuck with it, even if we want to change its name or something.